ContactDetailsApi/Startup.cs

```diff
@@ -167,8 +171,8 @@ private static void RegisterUseCases(IServiceCollection services)
 
         private static void RegisterFactories(IServiceCollection services)
         {
-            services.AddScoped<V1.Factories.ISnsFactory, V1.Factories.ContactDetailsSnsFactory>();
-            services.AddScoped<V2.Factories.ISnsFactory, V2.Factories.ContactDetailsSnsFactory>();
+            services.AddScoped<V1.Factories.Interfaces.ISnsFactory, V1.Factories.ContactDetailsSnsFactory>();
```

Comment:
could possibly save some characters with a using

Reply:
yeah, but its startup. I think its better to be explicit here.